ChangeLog

```diff
@@ -7,6 +7,8 @@ Bugfix
      invalidated keys of a lifetime of less than a 1s. Fixes #1968.
    * Fix failure in hmac_drbg in the benchmark sample application, when
      MBEDTLS_THREADING_C is defined. Found by TrinityTonic, #1095
+   * Fix memory leak and freeing without initialization in the example
```

Comment:
Do we add ChangeLog entries for the sample applications?

Reply:
For this one we do, because it was breaking a rather obvious test.